docs/dev/guidelines/database.rst

```diff
@@ -0,0 +1,236 @@
+**********************
+Database Relationship
```

Comment:
```suggestion
Database Relationships
```
maybe use the plural here as you´re talking about relationships in general and not just one?